src/tools/perf/perftest_params.c

```diff
@@ -74,6 +74,8 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -i <offset>    distance between consecutive scatter-gather entries (%zu)\n",
                                 ctx->params.super.iov_stride);
     printf("     -l             use loopback connection\n");
+    printf("                    there is no server process in this case\n");
+    printf("                    a server hostname in a client cmd is not allowed\n");
```

Comment:
use loopback connection. in this case, the process will communicate with itself, so passing server hostname is not allowed.